src/python/turicreate/toolkits/_model.py

```diff
@@ -132,6 +132,7 @@ def load_model(location):
                     return cls(model)
 
                 if name=='one_shot_object_detector' and OD_USE_CPP:
+                    import turicreate.toolkits.libtctensorflow
```

Comment:
Just curious why does this solve the issue reported?

Reply:
It solves the first part actually, which the break that reports no compute context. The TensorFlow compute context which was designed using PyBind needs to be linked at runtime. Without this import, the shared library is not linked and the compute context will not be found.